api/dto/target.go

```diff
@@ -157,11 +163,11 @@ func checkExpression(expression parser.Expr, ttl time.Duration, isRemote bool) *
 			continue
 		}
 
-		if badFunc := checkExpression(argument, ttl, isRemote); badFunc != nil {
+		if badFunc := checkFuncExpression(argument, ttl, isRemote); badFunc != nil {
```

Comment:
"checkFuncExpression" seems salt is salty

we check parsed expression, yea? and lets leave old name

Reply:
ok, I leave old name, because `if` will be in `checkExpression`, so name `checkFuncExpression` has no sense